<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Geocoding</class>
 <widget class="QWidget" name="Geocoding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地理编码</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/geocoding.png</normaloff>:/QtWidgetsApplication3/bin/icon/geocoding.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>干涉产品地理编码</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>选择工程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_project1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>选择节点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_node1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>目标节点名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dstnode1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="7,3">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>SAR图像地理编码</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>选择工程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_project2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>选择节点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_node2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>目标节点名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dstnode2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,2,3,2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>距离向多视</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_multi_rg"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>方位向多视</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_multi_az"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="7,3">
         <item>
          <widget class="QProgressBar" name="progressBar_2">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox_2">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
